test/integration/default/verify/controls/aws_s3_bucket_object.rb

```diff
@@ -0,0 +1,32 @@
+fixtures = {}
+[
+  's3_bucket_name',
+].each do |fixture_name|
+  fixtures[fixture_name] = attribute(
+    fixture_name,
+    default: "default.#{fixture_name}",
+    description: 'See ../build/s3.tf',
+  )
+end
+
+control 'aws_s3_bucket_object public file test' do
+  #------------------- Exists / Permissions / public file  -------------------#
+  describe aws_s3_bucket_object(name: fixtures['s3_bucket_name'], key: 'public-pic-read.jpg') do
```

Comment:
Please separate these into a group for doing searches and a group for checking each property and matcher.

Please add tests looking for objects that do not exist.